lib/bap/bap.mli

```diff
@@ -5284,6 +5297,12 @@ module Std : sig
     (** tags a section  *)
     val section : string tag
 
+    (** tags a code region  *)
+    val code_region  : code_region tag
+
+    (** tags a data region  *)
+    val data_region  : data_region tag
```

Comment:
We can also represent it with `unit tag` or `bool tag` (if we want to discriminate between writable and non-writable)